README.md

```diff
@@ -122,7 +125,7 @@ module "postgresql" {
                   "Action": [
                     "rds-db:connect"
                   ],
-                  "Resource": "arn:aws:rds-db:${module.eks_cluster.region}:${var.account_id}:dbuser:${var.aurora_cluster_name}/${var.aurora_irsa_username}"
+                  "Resource": "arn:aws:rds-db:${local.eks_cluster_region}:${var.account_id}:dbuser:${var.aurora_cluster_name}/${var.aurora_irsa_username}"
```

Comment:
just a suggestion whether local should be defined somewhere?
Feels a bit out of place in just the snippet as it's nowhere mentioned in the Readme.